### ISSUE TITLE
The cell reference  of table block is not announced by screen reader .

### ISSUE THREAD

**Issue by nsingh2001** [nsingh2001/Addpage-WP#2]
UAT 2022-09
Windows 10
Chrome Version 100.0.4896.88 (Official Build) (64-bit)
NVDA-2022
Repro step
1. open URLhttps://test-wp-manish.azurewebsites.net/wp-admin/ 
2. go to Page link and then Add new link.
3. Tab to block inserter and insert table block.
4. Go to table block and then select column and row and then create table.
5. Finally, go to table cell by using of down arrow and check cell position by using arrow key.
Expected Result:
The cell reference should be announce by screen reader like cell row and column.
Actual Result :
The screen reader announced only body cell.
Applicable to WCAG success criteria  :
https://www.w3.org/TR/WCAG21/#name-role-value
Attached  VIDEO FILE
[Cell Reference  of table.webm](https://user-images.githubusercontent.com/102017896/190376374-d12fa276-7161-4ea7-8649-c6b0afbf003b.webm)
